SU2_CFD/src/solver_direct_heat.cpp

```diff
@@ -235,11 +234,10 @@ CHeatSolverFVM::CHeatSolverFVM(CGeometry *geometry, CConfig *config, unsigned sh
     }
   }
 
-    for (iPoint = 0; iPoint < nPoint; iPoint++)
-      if (flow)
-        node[iPoint] = new CHeatFVMVariable(config->GetTemperature_FreeStreamND(), nDim, nVar, config);
-      else
-        node[iPoint] = new CHeatFVMVariable(Temperature_Solid_Freestream_ND, nDim, nVar, config);
+  if (flow)
+    node = new CHeatFVMVariable(config->GetTemperature_FreeStreamND(), nPoint, nDim, nVar, config);
```

Comment:
So all adaptions have been made for the heat solver already :+1:

Reply:
For all solvers and features that compile by default and are under regression.